deploy/deploy_app_server.yml

```diff
@@ -17,6 +17,8 @@
     - pycharm
     - nginx
     - nodejs
+    - redis
+    - scala
```

Comment:
@olivermel Looks like wires got crossed here, this PR doesn't include a `scala` role, so trying to build would fail.